website/content/en/guides/level-up/e2e-ack-guide.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Ensure data is always successfully delivered
+short: End-to-end acknowledgement
+description: Learn how to use end-to-end acknowledgement
+author_github: https://github.com/barieom
+domain: delivery
+weight: 5
```

Comment:
<img width="787" alt="image" src="https://user-images.githubusercontent.com/11903873/157560843-cc58581d-38a7-4511-937b-f8147a29accb.png">

This might need to be changed, or the others. I assume that's driving the guide numbering